definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,58 @@
+## Various latitude/longitude properties
+
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
+
+These values are machine-generated math centroids for a record's geometry.
+
+### intersection
+
+Find out more about the `intersection` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/intersection).
+
+The `intersection:latitude` and `intersection:longitude` represent the intersection location for a given record.
+
+### lbl
+
+Find out more about the `lbl` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/lbl).
+
+The `lbl:latitude` and `lbl:longitude` represent the label location for a given record.
+
+### local
+
+Find out more about the `local` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/local).
+
+The `local:latitude` and `local:longitude` represent what the "local" centroid for a "place" is. 
+
+### nav
+
+Find out more about the `nav` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/nav).
+
+The `nav:latitude` and `nav:longitude` represent the centroid used for a navigation endpoint.
```

Comment:
The `nav:latitude` and `nav:longitude` represent the centroid used for turn-by-turn navigation and "snapped" to the nearest road to the record's "entrance" (like the main entrance to a museum). For a locality this could be city hall or the center of downtown, or the road intersection of the largest roads in the place.